src/components/TeamMember/TeamMember.tsx

```diff
@@ -39,29 +39,41 @@ const TeamMember = ({ data }: { data: ITeamMember }) => {
     { name: "lattes", href: lattes, icon: "lattes", size: 18 },
   ];
 
+  const handleClick = () => {
+    const firstMedia = socialMedias.find((media) => media.href);
+    if (firstMedia) {
+      window.open(firstMedia.href);
+    }
+  };
+
```

Comment:
Instead of using a handleClick function to redirect the user to a link, it's best if we change the AvatarCircle component from a div to a <a> (link tag) and add the 

```js
socialMedias.find((media) => media.href).href
```

to it. I'm gonna comment this on the AvatarCircle component

```suggestion
  const firstMedia = socialMedias.find((media) => media.href);
```